pages/docs/user-docs/action-build.md

```diff
@@ -1,303 +1,185 @@
+The target defines the method that `build` uses to create the container. It can be one of the following:
 
+- URI beginning with **shub://** to build from Singularity Hub
+- URI beginning with **docker://** to build from Docker Hub
+- path to a **existing container** on your local machine
+- path to a **directory** to build from a sandbox
+- path to an **archive** in .tar or compressed .tar.gz format
+- path to a **[Recipe file](docs-recipes)**
 
-{% include toc.html %}
+In addition `build` can produce containers in three different formats.  Formats types can be specified by passing the following options to build.  
 
-## Bases
-You have many options for the base of your build. See the linked page for each for more details, or read the [Usage](#usage) section below for a summary.
+- compressed read-only **squashfs** file system suitable for production (default)
+- writable **ext3** file system suitable for interactive development (`--writable` option)
+- writable **(ch)root directory** called a sandbox for interactive development (`--sandbox` option)
 
-### Singularity Recipes
-For a reproducible container, the recommended practice is to build by way of a Singularity recipe file. This also makes it easy to add files, environment variables, and install custom software, and still start from your base of choice (e.g., Docker). The absolute minimum required for a recipe is a base, and here are your options:
+Because `build` can accept an existing container as a target and create a container in any of these three formats you can convert existing containers from one format to another.
 
+Some things need root...
 
-**Singularity Hub**
-```
-Bootstrap: shub
-From: vsoch/hello-world
-```
+## Downloading a pre-built container from Singularity Hub
+You can use the `build` command to download a container from Singularity Hub. 
 
-[**Docker**](/docs-docker)
 ```
-Bootstrap: docker
-From: tensorflow/tensorflow:latest
-IncludeCmd: yes # Use the CMD as runscript instead of ENTRYPOINT
+$ singularity build lolcow.simg shub://GodloveD/lolcow
 ```
 
-[**YUM/RHEL**](/build-yum)
-```
-Bootstrap: yum
-OSVersion: 7
-MirrorURL: http://mirror.centos.org/centos-%{OSVERSION}/%{OSVERSION}/os/$basearch/
-Include: yum
-```
+The first argument (`lolcow.simg`) specifies a path and name for your container.  The second argument (`shub://GodloveD/lolcow`) gives the Singularity Hub URI from which to download.
 
-[**Debian/Ubuntu**](/build-debootstrap)
-```
-Bootstrap: debootstrap
-OSVersion: trusty
-MirrorURL: http://us.archive.ubuntu.com/ubuntu/
-```
+But default the container will be converted to a compressed, read-only squashfs file.  If you want your container in a different format use the `--writable` or `--sandbox` options.
 
-[**Self**](/build-self)
-```
-Bootstrap: self
-```
+## Downloading a pre-built container from Docker Hub
+You can use `build` to download layers from Docker Hub and assemble them into Singularity containers. 
 
```

Comment:
This is still misleading. The container is not pre built. It is layers from Docker Hub that are assembled at build time to make a container. These subtle details are important and we need to be clear in the choice of words.

Reply:
How about download an "existing" container?